Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1

```diff
@@ -32,6 +32,17 @@ $gsi = $null
 #   Main body script
 #
 #######################################################################
+
+# Read parameters
```

Comment:
if you add TestParams parsing directly to Main, then this section is no longer required.
Please check sample code here:
https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/SETUP-Remove-NVME-Disk.ps1#L48

Reply:
Thank you for your guidance, new method is very convenient.